Make root path configurable from outside (#217)
elixir-nx/livebook: README.md

  * LIVEBOOK_SECRET_KEY_BASE - sets a secret key that is used to sign and encrypt the session and other payloads used by Livebook. Must be at least 64 characters long and it can be generated by commands such as: `openssl rand -base64 48`. Defaults to a random secret on every boot.

>   * LIVEBOOK_ROOT_PATH - the root path to use on file dialogs.

Reviewer: We use it on the homepage too =)

```suggestion
  * LIVEBOOK_ROOT_PATH - the root path to use for file selection.
```
